experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -2076,6 +2076,44 @@ it("enables save buttons when not archived", async () => {
   expect(screen.getByTestId("next-button")).toBeEnabled();
 });
 
+it("hides enrollment period when experiment is a rollout", async () => {
+  render(
+    <Subject
+      experiment={{
+        ...MOCK_ROLLOUT,
+        application: NimbusExperimentApplicationEnum.DESKTOP,
+        channel: NimbusExperimentChannelEnum.NIGHTLY,
+        status: NimbusExperimentStatusEnum.DRAFT,
+        isRollout: true,
+        firefoxMinVersion: NimbusExperimentFirefoxVersionEnum.FIREFOX_107,
+      }}
+    />,
+  );
+
+  await waitFor(() => {
+    expect(screen.queryByTestId("proposedEnrollment")).not.toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.queryByTestId("proposedEnrollment")).toBeNull();
```

queryByTestId returns null if not found, so we're testing if `null` will be in the document. It will never be true, but semantically it makes more sense to check for null in the return value.